.github/workflows/ci.yml

```diff
@@ -89,6 +89,8 @@ jobs:
             otp: '24.0'
           - elixir: '1.13.0'
             otp: '24.1'
+          - elixir: '1.14.0'
+            otp: '25.0'
```

Comment:
All the practice exercises have a comment in `mix.exs` with `# elixir: "~> 1.8"`. On one hand, it would be great to update that... on the other hand, it will cause everyone to have to update their exercise just for this new comment 🤦. I left it untouched.

Reply:
fair enough 🤷 
Let's reconsider next time we have serious friction with 1.8 :)

Reply:
I mean, I did delete the CI checks on 1.8 in [`98bc90b` (#1191)](https://github.com/exercism/elixir/pull/1191/commits/98bc90bed5f1b01f458219729ac00879ffa202a3). It's just that this comment is left in every exercise.